### ISSUE TITLE
Skip to main content link missing for section 1 intro

### ISSUE THREAD

**Issue by lauraponce** [truetandem/e-QIP-prototype#2627]

**Comment by Jay-Neighbours-I72** [comment]
Link is there now but not working. Not working is captured in this issue: #2620
